cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -562,6 +562,66 @@
     </description>
   </property>
 
+  <property>
+    <name>app.program.terminator.interval.secs</name>
+    <value>300</value>
+    <description>
+      Interval in seconds of how often the program terminator thread will run;
+      this value should be greater than 0
+    </description>
+  </property>
+
+  <property>
+    <name>program.run.status.monitor.service.retry.policy.base.delay.ms</name>
```

Comment:
This is a very long name. I believe we already have a retry policy for runtime monitor, can we just use that?